src/main/java/hudson/model/ExternalRun.java

```diff
@@ -92,15 +98,19 @@ private void setCharset(String c) { // JENKINS-14107
      * <p>
      * The format of the XML is:
      *
-     * <pre><xmp>
-     * <run>
-     *  <log>...console output...</log>
-     *  <result>exit code</result>
-     * </run>
-     * </xmp></pre>
+     * <pre>&lt;xmp&gt;
+     * &lt;run&gt;
+     *  &lt;log&gt;...console output...&lt;/log&gt;
+     *  &lt;result&gt;exit code&lt;/result&gt;
+     * &lt;/run&gt;
+     * &lt;/xmp&gt;</pre>
+     *
+     * @param in    Log file referenc
+     * @throws IOException
      */
     @SuppressWarnings({"Since15"})
     @IgnoreJRERequirement
+    @SuppressFBWarnings
```

Comment:
:ant: you should specify the actual warnings you are suppressing, and give a `justification` too.